momento-sdk/src/intTest/java/momento/sdk/BaseTestClass.java

```diff
@@ -1,35 +1,59 @@
 package momento.sdk;
 
 import java.time.Duration;
+import java.util.UUID;
 import momento.sdk.auth.CredentialProvider;
 import momento.sdk.config.Configurations;
+import momento.sdk.responses.cache.control.CacheCreateResponse;
+import momento.sdk.responses.cache.control.CacheDeleteResponse;
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 
 public class BaseTestClass {
-  public static final Duration FIVE_SECONDS = Duration.ofSeconds(5);
+  protected static final Duration DEFAULT_TTL_SECONDS = Duration.ofSeconds(60);
+  protected static final Duration FIVE_SECONDS = Duration.ofSeconds(5);
+  protected static CredentialProvider credentialProvider;
 
-  public static final CredentialProvider credentialProvider =
-      CredentialProvider.fromEnvVar("TEST_AUTH_TOKEN");
+  protected static CacheClient cacheClient;
+  protected static String cacheName;
 
   @BeforeAll
   static void beforeAll() {
-    if (System.getenv("TEST_AUTH_TOKEN") == null) {
-      throw new IllegalArgumentException(
-          "Integration tests require TEST_AUTH_TOKEN env var; see README for more details.");
-    }
-    if (System.getenv("TEST_CACHE_NAME") == null) {
-      throw new IllegalArgumentException(
-          "Integration tests require TEST_CACHE_NAME env var; see README for more details.");
+    credentialProvider = CredentialProvider.fromEnvVar("TEST_AUTH_TOKEN");
```

Comment:
Is it worth changing TEST_AUTH_TOKEN to MOMENTO_API_KEY in this pr? I thought we were slowly changing to that.

Reply:
I'm going to change the CI / env vars in a separate PR. This one is pretty busy already.